crates/driver/src/infra/solver/mod.rs

```diff
@@ -64,6 +64,21 @@ pub enum Liquidity {
     Skip,
 }
 
+#[derive(Clone, Copy, Debug)]
+pub struct Timeouts {
+    /// Maximum time allocated for http request/reponse to propagate through
+    /// network.
+    pub http_delay: chrono::Duration,
+    /// Maximum time allocated for processing the solutions received from
+    /// solvers, in percentage of total driver deadline, used for /solve
+    /// endpoint.
+    pub solve_competition_time: util::Percent,
```

Comment:
no longer really a time, but a share (of the total remaining time)

Reply:
Fixed.